### PR TITLE
Shopify: Add app.json plus logo

### PR DESCRIPTION
**Pull request by AndersLarsenMicrosoft**

### CODE REVIEW

**Review by JesperSchulz — APPROVED** [review]

File: Apps/W1/ShopifyConnector/app/app.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "id":  "ec255f57-31d0-4ca2-b751-f2fa7c745abb",
+    "name":  "Connect to Shopify",
+    "publisher":  "Microsoft",
+    "brief":  "To be updated",
+    "description":  "To be updated",
+    "version":  "20.0.0.0",
```
Comment (JesperSchulz) [inline Apps/W1/ShopifyConnector/app/app.json:7]: This PR is for version 20, while the GitHub branch is for version 19.3. This branch can hence to get merged as it is right now. But as this is a working branch, it should get merged. The PR is not ready.

**Review by JesperSchulz — CHANGES_REQUESTED** [review]
PR not ready for merge. Code missing. PR will be kept open for further contributions.

**Comment by AndersLarsenMicrosoft** [comment]
it is a working branch for now

**Comment by AndersLarsenMicrosoft** [comment]
working branch for now

**Pull request closed without merging**